doc/release-notes/release-notes-0.20.2.md

```diff
@@ -0,0 +1,180 @@
+by
+  the `listbanned` RPC. That RPC also no longer reports the `ban_reason`
+  field, as `"manually added"` is the only remaining option.
+
+- - Discouragement cannot be removed with the `setban remove` RPC command.
+  If you need to remove a discouragement, you can remove all
+discouragements by
+  stop-starting your node.
+
+Notification changes
+- --------------------
+
+`-walletnotify` notifications are now sent for wallet transactions that are
+removed from the mempool because they conflict with a new block. These
+notifications were sent previously before the v0.19 release, but had been
+broken since that release (bug
+[#18325](https://github.com/bitcoin/bitcoin/issues/18325)).
+
+PSBT changes
+- ------------
+
+PSBTs will contain both the non-witness utxo and the witness utxo for segwit
+inputs in order to restore compatibility with wallet software that are now
+requiring the full previous transaction for segwit inputs. The witness utxo
+is still provided to maintain compatibility with software which relied on
+its
+existence to determine whether an input was segwit.
+
+0.20.2 change log
+=================
+
+### P2P protocol and network code
+
+- - #19620 Add txids with non-standard inputs to reject filter (sdaftuar)
+- - #20146 Send post-verack handshake messages at most once (MarcoFalke)
+
+### Wallet
+
+- - #19740 Simplify and fix CWallet::SignTransaction (achow101)
+
+### RPC and other APIs
+
+- - #19836 Properly deserialize txs with witness before signing (MarcoFalke)
+- - #20731 Add missing description of vout in getrawtransaction help text
+(benthecarman)
+
+### Build system
+
+- - #20142 build: set minimum required Boost to 1.48.0 (fanquake)
+- - #20298 use the new plistlib API (jonasschnelli)
+- - #20880 gitian: Use custom MacOS code signing tool (achow101)
+- - #22190 Use latest signapple commit (achow101)
+
+### Tests and QA
+
+- - #19839 Set appveyor vm version to previous Visual Studio 2019 release.
+(sipsorcery)
+- - #19842 Update the vcpkg checkout commit ID in appveyor config.
+(sipsorcery)
+- - #20562 Test that a fully signed tx given to signrawtx is unchanged
+(achow101)
+
+### Miscellaneous
+
+- - #19192 Extract net permissions doc (MarcoFalke)
+- - #19777 Correct description for getblockstats's txs field (shesek)
+- - #20080 Strip any trailing / in -datadir and -blocksdir paths (hebasto)
+- - #20082 fixes read buffer to use min rather than max (EthanHeilman)
+- - #20141 Avoid the use of abs64 in timedata (sipa)
+- - #20756 Add missing field (permissions) to the getpeerinfo help
+(amitiuttarwar)
+- - #20861 BIP 350: Implement Bech32m and use it for v1+ segwit addresses
+(sipa)
+- - #22124 Update translations after closing 0.20.x on Transifex (hebasto)
+- - #21471 fix bech32_encode calls in gen_key_io_test_vectors.py (sipa)
+- - #22837 mention bech32m/BIP350 in doc/descriptors.md (sipa)
+
+Credits
+=======
+
+Thanks to everyone who directly contributed to this release:
+
+- - Aaron Clauson
```

Comment:
zero Diff for 21.2 but the 20.2 release notes being added here have formatting changes differing from https://raw.githubusercontent.com/bitcoin/bitcoin/v0.20.2/doc/release-notes.md

for example, why the double dashes here?

Reply:
Copied from: https://lists.linuxfoundation.org/pipermail/bitcoin-core-dev/2021-October/000106.html. Can update.